docs/images.coffee.md

```diff
@@ -33,6 +33,12 @@ Here is an example showing some of these options.
     # Scale the image
     doc.image('images/test.jpeg', 320, 280, scale: 0.25)
        .text('Scale', 320, 265)
+
+    # Fit the image in the dimensions, and center it both horizontally and vertically
+    doc.image('images/test.jpeg', 320, 15, fit: [100, 100], align: 'center', valign: 'center')
```

Comment:
The new example will hide the previous one, but I think there is enough space on the right:
```
doc.image('images/test.jpeg', 430, 15, fit: [100, 100], align: 'center', valign: 'center')
   .rect(430, 15, 100, 100)
   .stroke()
   .text('Centered', 430, 0)
```
![image](https://cloud.githubusercontent.com/assets/17966696/19149590/45c8f242-8bcb-11e6-97ce-a0946cfdad7f.png)